.travis.yml

```diff
@@ -9,107 +10,59 @@ install:
   - pip install -r requirements.txt -r test-requirements.txt
   - pip install -U .
   - pip freeze
+before_script:
```

Comment:
any reason to note move this to the installation step?

Reply:
No, I tried to make minimal changes, and kept this part.  I will move it to install.